### DISCUSSION TITLE
Mask Number removes comma?

### DISCUSSION THREAD

**Discussion by Basti271** [uNmAnNeR/imaskjs#959] [Q&A]
Using Nuxt 2 and vue-imask 7.1.3

The value outside of the Input is correct, but inside there is no comma shown. How I understood imask it should convert the dot to a comma inside the input? Am I doing sth wrong? Any help would be awesome!

```
<template>
  <div class="p-20">
    <input v-imask="mask" :value="value" />
    <div>Value: {{ value }}</div>
  </div>
</template>

<script>
import { IMaskDirective } from 'vue-imask'

export default {
  directives: {
    imask: IMaskDirective
  },

  data() {
    return {
      value: '12.56',
      mask: {
        mask: Number,
        radix: ',',
        mapToRadix: ['.']
      }
    }
  }
}
</script>
```

**Comment by uNmAnNeR** [comment]
seems like you pass unmasked value to (masked) value prop.
the initial value should be masked (`12,56`). The other solution could be to use unmasked value for input but it's not possible for directive. But it's possible for component/composable.
